server/src/routes/api/v1/reviewsRouter.js

```diff
@@ -0,0 +1,15 @@
+import express from 'express'
+import { Review } from "../../../models/index.js"
+
+const reviewsRouter = new express.Router()
+
+reviewsRouter.delete("/:id", async (req, res) => {
+  try {
+    await Review.query().deleteById(req.params.id)
```

Comment:
We wouldn't want someone to be able to delete someone else's review, and can check for that in the router here to see if the current user (`req.user`) is the same user that created this review to be deleted